<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_ModifyDormitory</class>
 <widget class="QWidget" name="Admin_ModifyDormitory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color:#3e6093;
}


QLineEdit{
border-radius:3px;
border:1px solid #3e6093;
background:#f2f6fd;
color:#3e6093;}
QComboBox{border-radius:3px;
border:1px solid #3e6093;
background:#f2f6fd;}


QComboBox::drop-down {
width:24px;
border-image:url(:/res/所有下拉框.png);
}  

QComboBox:hover{
background:#d9e4f5;
}

QComboBox:pressed{
background:#bac9e0;
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>370</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#f2f6fd;
border:1px solid #3e6093;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>修改宿舍信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#3e6093;</string>
    </property>
    <item>
     <property name="text">
      <string>梅苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>兰苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>竹苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>菊苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>桃苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>李苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>柳苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>桂苑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>荷苑</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>61</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>楼栋号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>141</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小寝号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>101</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大寝号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>21</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>宿舍区域</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>181</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>床号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#ffffff;
background-color:#3e6093;
border-radius:7px;
}

QPushButton:hover{
background:#2c466c;
}

QPushButton:pressed{
background:#253b5b;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#ffffff;
background-color:#3e6093;
border-radius:7px;
}

QPushButton:hover{
background:#2c466c;
}

QPushButton:pressed{
background:#253b5b;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
